extensions/data-plane/data-plane-signaling/data-plane-signaling-api-configuration/src/main/java/org/eclipse/edc/connector/api/signaling/configuration/SignalingApiConfigurationExtension.java

```diff
@@ -78,4 +91,16 @@ public void initialize(ServiceExtensionContext context) {
         webService.registerResource(webServiceConfiguration.getContextAlias(), new ObjectMapperProvider(jsonLdMapper));
         webService.registerResource(webServiceConfiguration.getContextAlias(), new JerseyJsonLdInterceptor(jsonLd, jsonLdMapper, SIGNALING_SCOPE));
     }
+
+    @Provider
+    public SignalingApiTransformerRegistry managementApiTypeTransformerRegistry() {
+        var factory = Json.createBuilderFactory(Map.of());
+
```

Comment:
Shouldn't the provider be in the `data-plane-signaling-transform` ? We would need  `SignalingApiTransformerRegistry` injected for also for the `signaling` client

Reply:
then it should the a provider in an extension on the client side IMO